### PR TITLE
Fixes deathgasping while being cloned

### PR DESCRIPTION
**Pull request by ghost**
:cl: More Robust Than You
fix: brain damage should no longer attempt to emote/say things while unconcious
/:cl:

This should, in theory, fix the cloning bug where brain damages causes you to lastwhisper before cloning is complete, killing you. And in general prevent brain damage from triggering last whispers with TAJARAN HAS WARES IF YOU HAZ COIN

### CODE REVIEW

File: code/modules/mob/living/carbon/human/life.dm [hunk 1]
```diff
@@ -66,7 +66,7 @@
 	else if(eye_blurry)			//blurry eyes heal slowly
 		adjust_blurriness(-1)
 
-	if (getBrainLoss() >= 60 && stat != DEAD)
+	if (getBrainLoss() >= 60 && stat != DEAD && stat != UNCONSCIOUS)
```
Comment (Cyberboss) [inline code/modules/mob/living/carbon/human/life.dm]: if (getBrainLoss() >= 60 && !stat)

**Review by Cyberboss — DISMISSED** [review]

**Comment by Repukan** [comment]
That's fucking funny tbh

File: code/modules/mob/living/carbon/human/life.dm [hunk 2]
```diff
@@ -66,7 +66,7 @@
 	else if(eye_blurry)			//blurry eyes heal slowly
 		adjust_blurriness(-1)
 
-	if (getBrainLoss() >= 60 && stat != DEAD)
+	if (getBrainLoss() >= 60 && !stat)
```
Comment (kevinz000) [inline code/modules/mob/living/carbon/human/life.dm]: stat == CONSCIOUS

Comment (Cyberboss) [inline code/modules/mob/living/carbon/human/life.dm]: Same difference you meme

**Pull request merged**